odb-compiler/src/parsers/db/Parser.y

```diff
@@ -813,8 +813,8 @@ loop_next_sym
   : SYMBOL %prec NO_HASH_OR_DOLLAR               { $$ = new AnnotatedSymbol(Symbol::Annotation::NONE, $1, driver->newLocation(&@$)); str::deleteCStr($1); }
   | SYMBOL '#'                                   { $$ = new AnnotatedSymbol(Symbol::Annotation::FLOAT, $1, driver->newLocation(&@$)); str::deleteCStr($1); }
   ;
-break
-  : BREAK                                        { $$ = new Break(driver->newLocation(&@$)); }
+exit
+  : EXIT                                         { $$ = new Exit(newloc()); }
```

Comment:
should be ```$$ = new Exit(driver->newLocation(&@$);```

Reply:
Ah yes just realised locally. Just fixed this but hitting a weird compile error so will push once I've resolved that.